dbms/src/Formats/FormatFactory.h

```diff
@@ -24,6 +25,12 @@ class WriteBuffer;
   */
 class FormatFactory final : public ext::singleton<FormatFactory>
 {
+public:
+    using ChunkCreator = std::function<bool(
```

Comment:
This needs a comment describing the requirements for the implementation of a file segmentation function, i.e. what it should do, what the arguments mean, when and in what context it is called etc.

Reply:
Add comment